### ISSUE TITLE
hertzg/rtl_433:alpine-3.12-20.11 linux/arm/v6 broken after adding SSL

### ISSUE THREAD

**Issue by mfugate1** [hertzg/rtl_433_docker#24]
Just wanted to let you know that the hertzg/rtl_433:alpine-3.12-20.11 ARM image no longer starts up with the latest changes that enabled SSL when building rtl_433 (I was running it on a raspi).

I switched to alpine-3.12-master which works fine, so I don't think this will block anyone. Maybe the easiest thing to do is just stop building this specific combination of alpine/rtl_433, or have SSL disabled for it.

```Error loading shared library libssl.so.48: No such file or directory (needed by /usr/local/bin/rtl_433)
Error loading shared library libcrypto.so.46: No such file or directory (needed by /usr/local/bin/rtl_433)
Error relocating /usr/local/bin/rtl_433: SSL_get_error: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_get_fd: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_write: symbol not found
Error relocating /usr/local/bin/rtl_433: SSLv23_client_method: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_free: symbol not found
Error relocating /usr/local/bin/rtl_433: SSLv23_server_method: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_CTX_use_certificate_chain_file: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_set_fd: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_CTX_set_verify: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_library_init: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_read: symbol not found
Error relocating /usr/local/bin/rtl_433: PEM_read_bio_DHparams: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_CTX_ctrl: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_new: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_CTX_use_certificate_file: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_shutdown: symbol not found
Error relocating /usr/local/bin/rtl_433: BIO_new_mem_buf: symbol not found
Error relocating /usr/local/bin/rtl_433: DH_free: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_accept: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_CTX_new: symbol not found
Error relocating /usr/local/bin/rtl_433: BIO_free: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_CTX_load_verify_locations: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_CTX_use_PrivateKey_file: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_CTX_set_cipher_list: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_connect: symbol not found
Error relocating /usr/local/bin/rtl_433: SSL_CTX_free: symbol not found
Error relocating /usr/local/bin/rtl_433: BIO_new_file: symbol not found```

**Comment by hertzg** [comment]
Thanks for reporting, this should be fixed by #22

**Comment by mfugate1** [comment]
Yep, can confirm it's fixed, thanks!
